idioms-xml/dormant/fireeye-pivy-report-with-indicators.xml

```diff
@@ -91,7 +91,7 @@
                     </FileObj:Hashes>
                 </cybox:Properties>
                 <cybox:Related_Objects>
-                    <cybox:Related_Object idref="fireeye:object-07e2d2f3-092d-436d-bbd6-60d2bdc36d43">
+                    <!-- <cybox:Related_Object idref="fireeye:object-07e2d2f3-092d-436d-bbd6-60d2bdc36d43">
```

Comment:
We don't we revert changes to this file to its initial state, and also have a /dormant directory under `idioms-xml`. Would that be reasonable?

Reply:
I was attempting to make the fireeye report make more sense.  The things I comments out were redundant, which makes it more difficult to determine if the patterns are good.  If/when I get back to it, I want to continue from that point.

Reply:
Sounds good.